### PR TITLE
Add user story question to agile-quiz.ts

### PR DESCRIPTION
**Pull request by gerhynes**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -75,34 +81,49 @@ const agileQuiz = [
...
+      "A simple description of a software feature from the perspective of a user or customer",
+    Distractor1:
+      "A simple description of a software feature from the perspective of a developer",
+    Distractor2:
+      "A simple description of a software feature from the perspective of the Scrum Master",
+    Distractor3:
+      "A simple description of a software feature from the perspective of the Product Owner",
+    Explanation:
+      "A user story is an informal, general explanation of a software feature written from the perspective of the end user.",
+    Link: "https://www.atlassian.com/agile/project-management/user-stories",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:125]: Whenever possible we always want to push freeCodeCamp resources.

Let's use this link instead

```suggestion
    Link: "https://www.freecodecamp.org/news/how-to-write-user-stories-epics-pesonas/",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/agile-quiz.ts (hunk 1 above)
Comment (gerhynes) [reply src/data/agile-quiz.ts]: Thanks for suggesting an appropriate freeCodeCamp resource, Jessica. I've updated the link now.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by gerhynes** [comment]
Thanks so much, Jessica. And thanks for all the work you're putting in maintaining this project, especially during Hacktoberfest.

**Pull request merged**